core/env/env_spec_helper.rb

```diff
@@ -0,0 +1,24 @@
+# Reserve variables.
```

Comment:
This file should be under`core/env/fixtures`, and to follow existing helper, be named `core/env/fixtures/classes.rb` and define a module ENVSpecs. No global methods should be defined in a helper file.